api/db-cdr/generate-cdr/libproject/devstart.rb

```diff
@@ -414,7 +414,7 @@ def publish_cdr_files(cmd_name, args)
       aw4_microarray_sources.each do |source_name, section|
         common.status("building manifest for '#{source_name}'")
         copy, output = build_manifests_for_aw4_section(
-          section, tier[:ingest_cdr_bucket], tier[:dest_cdr_bucket], op.opts.display_version_id, microarray_aw4_rows, output_manifest_path.call(source_name))
+          op.opts.project, section, tier[:ingest_cdr_bucket], tier[:dest_cdr_bucket], op.opts.display_version_id, microarray_aw4_rows, output_manifest_path.call(source_name))
```

Comment:
This is the only real change in this file, to pass the project to genomics_manifests.rb.